postprocessor/jdt-language-server/config_win/config.ini

```diff
@@ -1,10 +1,10 @@
 eclipse.p2.data.area=@config.dir/../p2
 eclipse.p2.profile=DefaultProfile
+osgi.bundles.defaultStartLevel=4
+eclipse.application=org.eclipse.jdt.ls.core.id1
+osgi.framework=file\:plugins/org.eclipse.osgi_3.17.100.v20211104-1730.jar
+osgi.framework.extensions=reference\:file\:org.eclipse.osgi.compatibility.state_1.2.500.v20210730-0750.jar
```

Comment:
We should move all these jars below into azsdkartifacts blob storage and convert them into regular maven dependencies (out of scope for this PR).